aten/src/ATen/NumericUtils.h

```diff
@@ -50,6 +50,39 @@ inline C10_HOST_DEVICE bool _isnan(at::BFloat16 val) {
   return at::_isnan(float(val));
 }
 
+// isfinite
+template <typename T,
+          typename std::enable_if<std::is_integral<T>::value, int>::type = 0>
+inline C10_HOST_DEVICE bool _isfinite(T val) {
+  return true;
+}
+template <typename T,
+          typename std::enable_if<std::is_floating_point<T>::value, int>::type = 0>
+inline C10_HOST_DEVICE bool _isfinite(T val) {
+#if defined(__CUDACC__) || defined(__HIPCC__)
+  return ::isfinite(val);
+#else
+  return std::isfinite(val);
+#endif
+}
+
+template <typename T,
+          typename std::enable_if<c10::is_complex<T>::value, int>::type = 0>
+inline bool _isfinite(T val) {
```

Comment:
The other functions here are annotated with C10_HOST_DEVICE - why isn't this one?

Reply:
Oops. Missed it. Thanks!